options/options.js

```diff
@@ -158,25 +158,25 @@ document.onclick = e => {
 
   connectButton.addEventListener('click', e => {
     if (validClick(e)) {
-      API.syncStart(cloud.value).catch(console.error);
+      API.sync.start(cloud.value).catch(console.error);
     }
   });
 
   disconnectButton.addEventListener('click', e => {
     if (validClick(e)) {
-      API.syncStop().catch(console.error);
+      API.sync.stop().catch(console.error);
     }
   });
 
   syncButton.addEventListener('click', e => {
     if (validClick(e)) {
-      API.syncNow().catch(console.error);
+      API.sync.now().catch(console.error);
```

Comment:
This looks pretty awkward to me. It makes me think that it returns a number or something because of `Date.now`.

Reply:
Good point. Guess I'll rename it to `runNow`

Reply:
Hmm, gonna be API.sync.syncNow to keep its internal name.